### DISCUSSION TITLE
Messy Code in the Log.js File

### DISCUSSION THREAD

**Discussion by shayanfpg9** [webpack/webpack#16401] [General]
## Feature request

**What is the expected behavior?**
use fewer lines of code to perform this operations

**What is motivation or use case for adding/changing the behavior?**
having a shorter code

**How should this be implemented in your opinion?**
change shouldLog function method,
changing default exported function method

**Are you willing to work on this yourself?**
yes

**Comment by alexander-akait** [comment]
What is `Log.js`?

**Reply by shayanfpg9** [reply]
the [log.js](https://github.com/webpack/webpack/blob/main/hot/log.js) in the Hot folder

**Reply by alexander-akait** [reply]
Feel free to send a PR, no need to create a new issue about it :+1:

**Reply by shayanfpg9** [reply]
ok, tnx
